pe-parser-library/src/parse.cpp

```diff
@@ -1616,6 +1631,8 @@ bool getSymbolTable(parsed_pe *p) {
       // Skip an unknown auxiliary record types
       offset += sizeof(uint8_t) * SYMTAB_RECORD_LEN * sym.numberOfAuxSymbols;
```

Comment:
I think this is fine because order of operations will extend the `SYMTAB_RECORD_LEN` from `uint16_t` to `size_t`, then do a similar extension of `uint8_t` to `size_t` for `sym.numberOfAuxSymbols`, but I think it would be good to have explicit casts, to mirror my above comments.